.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -0,0 +1,37 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["bug"]
+body:
+  - type: textarea
+    attributes:
+      label: Summary
+      description: |
+        A clear and concise description of the bug, including a minimal reproducible example.
+
+        Please include your full output with the complete error message.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Platform
+      description: What operating system and architecture are you using? (see `uname -orsm`)
+      placeholder: e.g., macOS 14 arm64, Windows 11 x86_64, Ubuntu 20.04 amd64
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Version
+      description: What version of janus-core are you using? (see `janus --version`)
+      placeholder: "e.g., janus-core version: 0.8.3"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Python version
+      description: What version of Python are you using? (see `python3 --version`)
```

Comment:
Also may not work with full virtualenvs like Conda/UV. Will need to ensure it's in the right environment

Reply:
Well, if UV're happy with it, I'm mostly happy with it. I'm not sure how often Python version will really be relevant either.

Reply:
We've seen a few bugs from (sub) dependencies that only affect specific Python version, so it can be useful.

Maybe more importantly, we've had several people try to install things with Python 3.6 etc, so if the "bug" is being unable to install anything, that becomes much easier to catch.